addons/mass_mailing/models/mass_mailing.py

```diff
@@ -57,6 +57,54 @@ def _compute_contact_nbr(self):
         for mailing_list in self:
             mailing_list.contact_nbr = data.get(mailing_list.id, 0)
 
+    @api.multi
+    def copy(self, default=None):
```

Comment:
The override of `copy` is useless. 

Just add the inverse field definition in the mail.mass_mailing.list model.

``` python
    contact_ids = fields.Many2many(
        'mail.mass_mailing.contact', 'mail_mass_mailing_contact_list_rel', 'list_id', 'contact_id',
        string='Mailing Lists')
```